build-scripts/for-linux/build-on-linux.bash

```diff
@@ -6,6 +6,19 @@
 # $4 - Paralllel threads
 # $5 - Source Dir. If not set then relative to the script dir
 
+#!/bin/bash
+
+get_oldest_java_path()
+{
+    if ! [ -x "$(command -v update-java-alternatives)" ]
+    then
+      echo 'update-java-alternatives is not installed'
+      JAVA_OLDEST_PATH=$(ls -d /etc/alternatives/java_sdk_*[0-9] | sort -V | head -n 1)
```

Comment:
Setting a variable inside a function is not an evident way.

Use `echo` and ``JAVA_OLDEST_PATH=``get_oldest_java_path`` instead.